scene/2d/physics/physics_body_2d.cpp

```diff
@@ -33,7 +33,7 @@
 #include "scene/scene_string_names.h"
 
 void PhysicsBody2D::_bind_methods() {
-	ClassDB::bind_method(D_METHOD("move_and_collide", "motion", "test_only", "safe_margin", "recovery_as_collision"), &PhysicsBody2D::_move, DEFVAL(false), DEFVAL(0.08), DEFVAL(false));
+	ClassDB::bind_method(D_METHOD("move_and_collide", "motion", "test_only", "safe_margin", "recovery_as_collision", "cancel_sliding"), &PhysicsBody2D::_move, DEFVAL(false), DEFVAL(0.08), DEFVAL(false), DEFVAL(true));
```

Comment:
I'd suggest adding `max_collisions` at the same time here as we can't do it later without breaking the order and alignment between 2D/3D

Reply:
My bad it isn't supported in the 2D server

Reply:
The thing is that `max_collisions` in 2D doesn't really exist. :thinking: :thought_balloon:

Reply:
Just saw your fast answer. Thanks!